.github/workflows/build.yml

```
@@ -20,67 +27,49 @@ jobs:
                   ref: ${{ github.head_ref }}
                   repository: ${{github.event.pull_request.head.repo.full_name || github.repository }}
-            - name: Install LLVM dependencies
+            - name: Install LLVM dependencies from the cache (Linux)
+              if: runner.os == 'Linux'
               uses: awalsh128/cache-apt-pkgs-action@latest
               with:
                   packages: llvm-20 llvm-20-dev libpolly-20-dev clang-20
                   version: 1.0
+            - name: Install LLVM dependencies from Homebrew (macOS)
+              if: runner.os == 'macOS'
+              run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:56: Double quote to prevent globbing and word splitting

(shellcheck)